### DISCUSSION TITLE
[Feature Request] - Delta 2 Max - Automated tasks

### DISCUSSION THREAD

**Discussion by g-force-be** [tolwi/hassio-ecoflow-cloud#190] [Ideas]
If possible via the API: activate / deactivate or even edit / create the Automated Tasks that are available in the app (Lab features). This would help to better manage the charging/discharing of the battery based on other devices/sensors in the home.

![20240110_143748130_iOS](https://github.com/tolwi/hassio-ecoflow-cloud/assets/47824024/a8b4ec12-2ede-47f1-a4a7-df51439ac2d2)

**Comment by Fyr42** [comment]
Their automated task is bugged to death. Mainly because at some hours, when battery is depleted, it's offline and the automation doesn't occur. Idem for the PowerStream which sleeps every night offline.
